### ISSUE TITLE
Firefox version?

### ISSUE THREAD

**Issue by superluig164** [wavenet-for-chrome/extension#90]
I'm sure this is a bit of a reach, but due to Chrome's impending removal of Manifest V2, I am beginning to start transitioning to Firefox.  However, this extension doesn't seem to have a similar equivalent on Firefox.  I'm not sure how simple it is to port this extension to Firefox, but is that in the cards?  Even if it's just this current version and never gets updated, it would help me a great deal with my transition, as most of my other extensions are available.  I'm sure I'm not the only person thinking this, but I thought I'd put it out there.  I use this extension liberally and it would be a huge loss for me to not be able to use it on Firefox.  Thank you for your consideration.

**Comment by pgmichael** [comment]
It would be a nice addition, however, at the time being this extension is not a main focus for me so it is unlikely for me to implement this.

Happy to review any PRs if someone every tries to add this though; If I can recall correctly (I haven't worked on this in a while) there are a few Chrome specific APIs/calls done that may be tricky to migrate.

**Comment by superluig164** [comment]
Fair enough.  Hopefully someone tries to do it, or maybe you randomly come across the motivation to try it sometime in the future.  Thank you for the quick reply.
